packages/circle-react-components/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "circle-react-components",
+  "version": "0.0.1",
+  "packageManager": "yarn@1.22.22",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ChainSafe/web3-circle-libs/"
+  },
+  "type":"module",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "engines": {
+    "node": ">=20.0.0"
+  },
+  "scripts": {
+    "dev": "storybook dev -p 6006",
+    "build:storybook": "storybook build",
+    "build": "rollup -c",
+    "lint": "eslint --ext .js,.jsx,.ts,.tsx --fix",
+    "prettier": "prettier --write \"src/**/*.{ts,tsx,js,mdx}\"",
+    "test": "vitest",
+    "test:cov": "vitest run --coverage",
```

Comment:
does all these script work? I don't see storybook config here for example. I think it's fine in the first version but storybook should live together with components as that's the way to preview them in a sandbox like environment.